Formula/rubberband.rb

```diff
@@ -30,15 +30,16 @@ class Rubberband < Formula
     depends_on "fftw"
     depends_on "gcc"
     depends_on "ladspa-sdk"
-    depends_on "openjdk"
     depends_on "vamp-plugin-sdk"
   end
 
   fails_with gcc: "5"
```

Comment:
We can compile `rubberband` with GCC 5 if we add a `#include <stddef.h>`, which may remove some direct `gcc` linkage/requirement in dependents.

However, this isn't that important as `gcc` is already in dependency tree due to `fftw --> open-mpi --> gcc` and we seem to be considering a different solution for dealing with Linux GCC usage.